src/Common/hooks/useFilters.tsx

```diff
@@ -38,7 +49,10 @@ export default function useFilters({ limit = 14 }: { limit?: number }) {
   };
   const removeFilter = (param: string) => removeFilters([param]);
 
-  useEffect(() => updateFiltersCache(qParams), [qParams]);
+  useEffect(
+    () => updateFiltersCache(qParams, primaryCacheBlacklist),
+    [qParams, primaryCacheBlacklist]
+  );
```

Comment:
This is not required when the useState is avoided

```suggestion
```

Reply:
so delete the whole useEffect ?

Reply:
yup